schema-derivatives-options.graphql

```diff
@@ -154,6 +160,9 @@ interface Protocol {
   " Current TVL (Total Value Locked) of the entire protocol "
   totalValueLockedUSD: BigDecimal!
 
+  " Current PCV (Protocol Controlled Value). Only relevant for protocols with PCV. "
+  protocolControlledValueUSD: BigDecimal
```

Comment:
Is there a reason we want this aside from making it work in subgraphs.xyz?

This is the case bc the options schema is not integrated in subgraphs.xyz yet. It is defaults to look at the perps schema. This is being fixed by Michael C though

Reply:
Without protocolControlledValueUSD, subgraphs.xyz complains the schema does not comply with generic schema. Now with protocolControlledValueUSD, subgraphs.xyz looks like it doesn't support options schema yet. Nevertheless, it is added to just make it works with subgraphs.xyz.

Reply:
@ishraq8 do we have a need for PCV in the options schema?

@corerouter What I am getting at is that we should not add it reactively just to make subgraphs.xyz happy. Whatever we decide, Michael C said he would work on adding the options schema into subgraphs.xyz, so we don't have to worry about that.

Reply:
I see. Just restore PCV part to previous versions before I made the changes at 7cee2193120a9b72b80495e89f048f6a5bcc7cb3 @dmelotik @ishraq8